TestAssets/NonRestoredTestProjects/DotnetAddP2PProjects/ValidRef/ValidRef.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>net451;netcoreapp1.0</TargetFrameworks>
+    <TargetFrameworks>net451;netcoreapp1.0;netstandard1.4</TargetFrameworks>
```

Comment:
I need to look through these tests. This seems like a smell to me. Are we building such convoluted test projects to cover a bunch of scenarios that they're not representative of any real customer scenario?